addons/block_code/ui/tooltip/tooltip.gd

```diff
@@ -22,3 +22,13 @@ func _ready():
 	bold_italics.set_base_font(italics)
 	bold_italics.set_variation_embolden(1.2)
 	add_theme_font_override(&"bold_italics_font", bold_italics)
+
+
+## Polyfill of Node.is_part_of_edited_scene(), available to GDScript in Godot 4.3+.
+static func node_is_part_of_edited_scene(node: Node) -> bool:
+	return Engine.is_editor_hint() && node.is_inside_tree() && node.get_tree().edited_scene_root && node.get_tree().edited_scene_root.get_parent().is_ancestor_of(node)
```

Comment:
When I wrap this incredibly long line, `gdformat` insists on changing it back...

Reply:
Even with parentheses?

Reply:
Yes, amazingly enough

Reply:
Oh right, it's because the maximum line length was set to 200 chars in e29f171f.